trinity/plugins/builtin/ethstats/ethstats_client.py

```diff
@@ -43,8 +43,8 @@ def __init__(
         self.websocket = websocket
         self.node_id = node_id
 
-        self.send_queue: asyncio.Queue[EthstatsMessage] = asyncio.Queue()
-        self.recv_queue: asyncio.Queue[EthstatsMessage] = asyncio.Queue()
+        self.send_queue: asyncio.Queue['EthstatsMessage'] = asyncio.Queue()
+        self.recv_queue: asyncio.Queue['EthstatsMessage'] = asyncio.Queue()
```

Comment:
It is the entire `'asyncio.Queue[EthstatsMessage]'` that would need quotes. But if you say that it works without those maybe just leave them out. That said, I gotta double check that because everywhere else we had to quote `asyncio.Queue[T]` so far.

Reply:
Was curious about why that happened, found following:

https://www.python.org/dev/peps/pep-0526/#runtime-effects-of-type-annotations

Since that declaration is inside the method it gets treatment as a local variable and type definition is not evaluated, so can be any value (even undefined). `mypy` gets type definitions from `typeshed` where `asyncio.Queue` is defined, so both runtime and `mypy` check work without quotes. If I move that declaration to a class level, it will get evaluated and will fail, so in that context it will need quotes.

Reply:
Ha! Great find! Thanks for investigating. Let's do it without quotes then.